orchestration/db/api.py

```diff
@@ -58,7 +95,10 @@ def get_service_definition(id='', context=None):
         else:
             query = session.query(models.ServiceDefinition).filter(
                 models.ServiceDefinition.id == id)
-    return None if not query else query.first()
+    if not query:
+        return None
+    else:
+        return get_query_res(query.all(), models.ServiceDefinition)
```

Comment:
Why don't we use query.first()? Else, in the caller you will have to do res[0]

Reply:
In the caller, result is the list of the dict for the db objects.